### PR TITLE
Copy rpms to the destination (fix #241)

### LINKED ISSUE #241: copy built rpms from `base.rpm_path` to user selected destination
depends on #238

### PR DESCRIPTION
**Pull request by regeciovad**

### CODE REVIEW

File: rpg/gui/wizard.py [hunk 1]
```diff
@@ -752,6 +752,8 @@ def buildRpm(self):
         arch = self.BuildArchEdit.currentText()
         distro = self.BuildDistroEdit.currentText()
         self.base.build_rpm_recover(distro, arch)
+        Command("cp " + str(self.base.rpm_path) + " " +
```
Comment (jsilhan) [inline rpg/gui/wizard.py]: base.rpm_path returns a list

File: rpg/gui/wizard.py [hunk 2]
```diff
@@ -752,6 +752,10 @@ def buildRpm(self):
         arch = self.BuildArchEdit.currentText()
         distro = self.BuildDistroEdit.currentText()
         self.base.build_rpm_recover(distro, arch)
+        packages = self.base.rpm_path
```
Comment (jsilhan) [inline rpg/gui/wizard.py]: why this extra line?

**Comment by jsilhan** [comment]
test this locally if it works, please

File: rpg/gui/wizard.py (hunk 2 above)
Comment (regeciovad) [inline rpg/gui/wizard.py]: Well, it's not necessary, but I prefer for cycle with local variable.

**Comment by regeciovad** [comment]
I tested it with actual master branch in gui with rpg-0.0.2-1. Rpm package wasn't created and rpm_path returned "Can't find '{}'! You need to call build_rpm first."

**Comment by jsilhan** [comment]
Merged, thanks.

**Pull request closed without merging**